### ISSUE TITLE
Pass values to def get_embed_info():

### ISSUE THREAD

**Issue by peya02** [lipinght/PBIEmbedRLSFlaskSample#4]
I have the following problem and first I would like to clarify how it works.
I have the def view_repor(id_repor): which in turn calls another function that what it does is make a query to a database and according to the user and the id of a powerbi report (previously passed) it outputs an id.

```
@app.route("/view_repor/<int:id_repor>")
def view_repor(id_repor):
    if not auth.get_user():
        return redirect(url_for("login"))
    user=auth.get_user()
    usuario=user['oid']
    
    global repor
    repor = obtener_report_por_id_user2(db,usuario,id_repor)
    return render_template("view_repor.html", repor=repor)
```

Well, so far there is no problem. I want to pass this id along with the report id to the following function def get_embed_info(): so that it launches
embed_info = PbiEmbedService().RLS_roles(app.config['WORKSPACE_ID'], report[2], user, report[3])
which is the def where I pass the correct parameters, both report id, roles, WORKSPACE_ID...
To pass the values at the moment I have it like this although I know it is not correct. I make a global variable from the previous result "global repor" and I take that variable in the get_embed_info() def as I show below.

```
@app.route('/getembedinfo', methods=['GET'])
def get_embed_info():
    user=auth.get_user()
    usuario=user['oid']
    reporte=repor
    config_result = Utils.check_config(app)
    if config_result is not None:
        return json.dumps({'errorMsg': config_result}), 500

    try:
        embed_info = PbiEmbedService().RLS_roles(app.config['WORKSPACE_ID'], reporte[2],usuario,reporte[3])
        return embed_info
        
    except Exception as ex:
        return json.dumps({'errorMsg': str(ex)}), 500
```

Well, after explaining how my app works, I explain the problem.
I tested the code on localhost against a database and it works perfectly, it takes the values, passes them correctly to def get_embed_info(): and also to PbiEmbedService().RLS_roles(app.config['WORKSPACE_ID'], report[2], user,report[3])

But when I upload the same app to a hosting (connected to its database also mysql) it no longer does the same thing. For some reason the value of the previous query that I passed in the global variable changes.

If repor[2] was worth 33 in repor = obtain_report_by_id_user2(db, user, repor_id)

When you get to the def get_embed_info(): it is worth 44

Basically it no longer pays attention to the result of the query but rather takes any random line from the database table.
